stories/list-grid/list-grid.stories.js

```diff
@@ -0,0 +1,1880 @@
+                                    <p>USA</p>
+                                </div>
+
+                                <a href="#" class="fd-link" tabindex="0">john_li@example.com</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div  class="fd-col fd-col--12 fd-col-md--6 fd-col-lg--4 fd-col-xl--3">
+                <div tabindex="0" class="fd-grid-list__item">
+                    <div class="fd-grid-list__item--body fd-grid-list__item--body--padding">
+                        <div class="grid-list-item-body-example">
+                            <span class="fd-avatar fd-avatar--s grid-list-item-body-example-avatar" role="presentation" aria-label="John Doe"></span>
+                            <div>
+                                <h4 class="fd-title fd-title--h4">John Li</h4>
+                                <p>Product Owner, Company B</p>
+
+                                <div>
+                                    <p>781 Main Street</p>
+                                    <p>Anytown, SD 57401</p>
+                                    <p>USA</p>
+                                </div>
+
+                                <a href="#" class="fd-link" tabindex="0">john_li@example.com</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+group.storyName = 'Group';
+
+group.parameters = {
+    docs: {
+        storyDescription: `
+If grouped, a group header is displayed above all items which belong to the corresponding group. The group header is not interactive.
+To create grid list group headers use \`fd-grid-list__group-header\` class.
+    `
+    }
+};
+
+export const states = () => `<div class="fd-grid-list">
+    <div class="fd-container">
+        <div class="fd-row">
+            <div class="fd-col fd-col--12">
+                <div class="fd-toolbar fd-toolbar--solid">
+                    <span>Products (420)</span>
+
+                    <span class="fd-toolbar__spacer"></span>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Sort">
+                        <i class="sap-icon--sort"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Filter">
+                        <i class="sap-icon--filter"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Group">
+                        <i class="sap-icon--group-2"></i>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <div class="fd-row">
+            <div  class="fd-col fd-col--12 fd-grid-list__group-header">
+                State: Unread
```

Comment:
This group header is not announced when you tab into the group. It should be read upon accessing the first grid item. The same applies for all header examples.

Reply:
fixed